File: 1019-squares-of-a-sorted-array/solution.py
class Solution:
    def sortedSquares(self, nums: List[int]) -> List[int]:
        ans = [x**2 for x in nums]
        l = 0
        r = len(ans)-1
        an = []
        
        while l <= r:
            if ans[r] > ans[l]:
                an.append(ans[r])
                r -= 1
            else:
                an.append(ans[l])
                l += 1
        return an[::-1]


        
            
        

File: 1019-squares-of-a-sorted-array/solution.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var sortedSquares = function(nums) {
    const ansarr = nums.map((val,indx)=>val*val)
    return ansarr.sort((a,b)=>a-b)
    
};
